spiel/st/m0t1fs_spiel_sns_repair_quiesce.sh

```diff
@@ -34,8 +34,8 @@ spiel_sns_repair_quiesce_test()
 
 	echo "Starting SNS repair testing ..."
 	for ((i=0; i < ${#files[*]}; i++)) ; do
-		touch_file $MOTR_M0T1FS_MOUNT_DIR/${files[$i]} ${unit_size[$i]}
-		_dd ${files[$i]} $((${unit_size[$i]} * 1024)) ${file_size[$i]}
+		touch_file "$MOTR_M0T1FS_MOUNT_DIR"/"${files[$i]}" "${unit_size[$i]}"
```

Comment:
Please find out if this is a better way:
touch_file "$MOTR_M0T1FS_MOUNT_DIR/${files[$i]}" "${unit_size[$i]}"

Reply:
done